.github/workflows/ci-ultra-tests.yml

```diff
@@ -40,7 +36,6 @@ jobs:
           python3.7 -m venv .venv
           . .venv/bin/activate
           pip install dill
```

Comment:
Why is this `dill` package required separately outside of `requirements.txt` and `setup.py` files?

Reply:
Hmm, that is correct. I think the issue was that the requirements file was not updated on my side, which include the dill module